tracer/missing-nullability-files.csv

```diff
@@ -149,9 +149,9 @@ src/Datadog.Trace/HttpOverStreams/HttpMessage.cs
 src/Datadog.Trace/HttpOverStreams/HttpRequest.cs
 src/Datadog.Trace/HttpOverStreams/HttpResponse.cs
 src/Datadog.Trace/HttpOverStreams/IHttpContent.cs
-src/Datadog.Trace/Iast/Iast.cs
-src/Datadog.Trace/Iast/ITaintedMap.cs
-src/Datadog.Trace/Iast/SourceType.cs
+src/Datadog.Trace/IAST/Iast.cs
+src/Datadog.Trace/IAST/ITaintedMap.cs
+src/Datadog.Trace/IAST/SourceType.cs
```

Comment:
tl;dr; git being case sensitive and windows being non-case sensitive is a PITA

so you should
- Make sure you don't have anything you need to commit etc (is `git status` empty?)
- Delete the folder `src/Datadog.Trace/IAST`
- run `git reset HEAD --hard`
- run `\tracer\build.ps1 CreateMissingNullabilityFile`

Then this file should be reverted again